integration_test/list_test.go

```diff
@@ -40,5 +40,41 @@ func TestKrewList(t *testing.T) {
 	if diff := cmp.Diff(eventualList, expected); diff != "" {
 		t.Fatalf("'list' output doesn't match:\n%s", diff)
 	}
-	// TODO(ahmetb): install multiple plugins and see if the output is sorted
+
```

Comment:
The other integration test files cleanly separate test cases into separate test functions. Please also follow that pattern here.

Reply:
Changed and i'll move it to another PR